Remotion/Web/Core/UI/Controls/WebMenuItem.cs

```diff
@@ -169,21 +177,20 @@ public string Category
 
     [PersistenceMode (PersistenceMode.Attribute)]
     [Category ("Appearance")]
-    [Description ("The text displayed in this menu item. Use '-' for a separator menu item. The value will not be HTML encoded.")]
+    [Description ("The text displayed in this menu item. Use '-' for a separator menu item.")]
     [NotifyParentProperty (true)]
     [DefaultValue ("")]
-    [AllowNull]
-    public string Text
+    public WebString Text
     {
       get { return _text; }
-      set { _text = value ?? string.Empty; }
+      set { _text = value; }
     }
 
     [DesignerSerializationVisibility (DesignerSerializationVisibility.Hidden)]
     [Browsable (false)]
     public bool IsSeparator
     {
-      get { return _text == c_separator; }
+      get { return _text.GetValue() == c_separator; }
```

Comment:
Should we make the c_separator into a WebString static instead of a string const and then we can do a compare here? Didn't check what that means for all the usages of c_separator, though.

Reply:
Ah, yes, in case someone has the idea of setting `-` with encoded. Hadn't thought of that but you probably can't prevent such nonsense. so, good choice!